ginga/rv/plugins/LineProfile.py

```diff
@@ -275,45 +303,55 @@ def _slice(self, naxes, mk):
 
         # For axis > 3
         for i in range(2, naxes):
-            slice_obj[i] = int(round(self.image.revnaxis[i-2] + 1))
+            if i != i_sel:
+                slice_obj[i] = 0  # int(round(self.image.revnaxis[i-2] + 1))
```

Comment:
> I think this is so that the plugin updates correctly when then the slice is changed through `MultiDim`...

I just tested this theory using an unmodified version of `LineProfile` and disproved it. When I play through the cube in `MultiDim`, the plot does not change. Also, for 3D cube, the value is always 1. So, I still don't understand what is the purpose of this complex calculation.